sqlglot/diff.py

```diff
@@ -0,0 +1,300 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from sqlglot.expressions import (
```

Comment:
as this gets bigger, i prefer to do from sqlglot import exp or import from sqlglot import expressions as exp

Reply:
Fixed